financial-services-accelerator/internal-apis/org.wso2.financial.services.accelerator.authentication.endpoint/src/main/webapp/libs/bootstrap_3.4.1/js/npm.js

```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
+require('../../js/transition.js')
+require('../../js/alert.js')
+require('../../js/button.js')
+require('../../js/carousel.js')
+require('../../js/collapse.js')
+require('../../js/dropdown.js')
+require('../../js/modal.js')
+require('../../js/tooltip.js')
+require('../../js/popover.js')
+require('../../js/scrollspy.js')
+require('../../js/tab.js')
+require('../../js/affix.js')
```

Comment:
lets add an EOF line here

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/110/commits/33337e150774c188bd8783e602cf48d83f28ff56